packages/lib-utils/src/app/api-discovery/index.ts

```diff
@@ -134,13 +136,13 @@ const getResources = async (
     },
     {},
   );
-  const all = _.flatten(
-    apiResourceData.groups.map<string[]>((group) =>
-      group.versions.map<string>((version) => `/apis/${version.groupVersion}`),
-    ),
-  )
-    .concat(['/api/v1'])
-    .sort((api) => (preferenceList.find((item) => api.includes(`/apis/${item}`)) ? -1 : 0));
+  const all = ['/api/v1'].concat(
```

Comment:
@karelhala I know this was an issue before your change, but we should consider removing the hard-coded version here (IE if we switch to version `V2` in the distant future). Not blocking.

Reply:
Which api we'd call for fetching the core models? We can add a constant by all means, just curious how'd you like to remove the version.

Reply:
I'm just suggesting we should have a singular location for the version of the core APIs - not that the API is changing or will change. For now this LGTM.